src/main/java/seedu/address/model/person/Person.java

```diff
@@ -62,6 +77,31 @@ public Set<Skill> getSkills() {
         return Collections.unmodifiableSet(skills);
     }
 
+    public String getMatch() {
+        return match;
+    }
+
+    /**
+     * Checks if this person has matched with a job.
+     *
+     * @param jobIdentifier A string that uniquely identify a job
+     */
+    public boolean hasMatched(String jobIdentifier) {
+        if (match == null) {
```

Comment:
Maybe instead of using null can use empty set?

Reply:
As discussed, I have changed the implementation of match variable with Optional<String>.